src/utils/configLoad.ts

```diff
@@ -112,11 +109,11 @@ export async function loadAndValidateConfig(configSource: string | undefined) {
 
   maybeInjectAnalyticsLibrary(config);
 
-  // TODO: refactor this once we have more input how integrations will be handled in dashboard
-  if (config?.integrations?.cleeng?.id && (config?.integrations?.jwp?.clientId || config?.integrations?.inplayer?.clientId)) {
+  if (config?.integrations?.cleeng?.id && config?.integrations?.jwp?.clientId) {
```

Comment:
@dbudzins we could use it here

```
const {cleeng, jwp} = config?.integrations || {};

  if (cleeng?.id && jwp?.clientId) {
  ...
```

Reply:
I tend to err on the side of fewer temp variables where possible. The chaining doesn't bother me in this case.